### PR TITLE
`NugetITCase` with `TestDeployment`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #855
Remade `NugetITCase` to use `TestDeployment`

### CODE REVIEW

**Comment by genryxy** [comment]
@olenagerasimova please check failure with `.pdd`

File: src/test/java/com/artipie/nuget/NugetITCase.java [hunk 1]
```diff
@@ -23,120 +23,43 @@
...
-        MatcherAssert.assertThat(
-            this.pushPackage(),
-            new StringContains("Your package was pushed.")
-        );
-    }
-
-    @Test
-    @Timeout(30)
-    @Disabled
-    void shouldInstallPushedPackage() throws Exception {
```
Comment (genryxy) [inline src/test/java/com/artipie/nuget/NugetITCase.java:119]: @olenagerasimova also one concern about this test case. Probably remain `@todo` to add installation test or rewrite it

**Comment by olenagerasimova** [comment]
@genryxy thanks, the test method both push and installs the package, I've corrected method name and do not think another test is required here. Also, removed unused class and corrected PDD config.

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, got it

**Pull request merged**